backends/ebpf/ebpfOptions.cpp

```diff
@@ -23,4 +23,7 @@ EbpfOptions::EbpfOptions() {
         registerOption("--emit-externs", nullptr,
                 [this](const char*) { emitExterns = true; return true; },
                 "[ebpf back-end] Allow for user-provided implementation of extern functions.");
+        registerOption("--trace", nullptr,
```

Comment:
another alternative is to always emit them but to guard them with a macro `#if TRACE` which can be enabled when compiling the resulting C. But this is fine.